webapp-django/crashstats/base/views.py

```diff
@@ -27,5 +26,5 @@ def handler404(request):
             'path': request.path,
             'query_string': request.META.get('QUERY_STRING'),
         }, status=404)
-    context = {'product': settings.DEFAULT_PRODUCT}
+    context = {}
     return render(request, '404.html', context, status=404)
```

Comment:
You can rewrite this as ``return render(request, '404.html', status=404)`` (and same for the ``handler500`` view).